app/src/main/java/algorithm/study/pureun/programmers/Prg_120822.java

```diff
@@ -0,0 +1,33 @@
+package algorithm.study.pureun.programmers;
+
+/**
+ * 프로그래머스 - 문자열 뒤집기
+ *
+ * @link https://school.programmers.co.kr/learn/courses/30/lessons/120822
+ */
+public class Prg_120822 {
+
+	public static void main(String[] args) {
+		Prg_120822 sol = new Prg_120822();
+
+		String ans = sol.solution("jaron");
+		System.out.println(ans);
+
+	}
+	
+    // String타입을 char로 변환하면 한글자씩의 배열로 변환되나봄
+	// char = 문자1개를 배열로하면 ==String이지 
+	public String solution(String my_string) {
+		String answer = "";
+		
+		char[] ch = new char[my_string.length()];
+		for(int i = 0; i < my_string.length(); i++) {
+			ch[i] = my_string.charAt(my_string.length()-1-i);
+		}
+		String str = new String(ch);
+		answer = str;
+		
+		return answer;
+	}
```

Comment:
```suggestion
    public String solution(String my_string) {
        
        char[] ch = my_string.toCharArray();
        for(int start = 0; start < ch.length / 2; start++) {
            int end = ch.length - start - 1;
            swap(ch, start, end);
        }
        
        return new String(ch);
    }
    
    private void swap(char[] ch, int i, int j) {
        char temp = ch[i];
        ch[i] = ch[j];
        ch[j] = temp;
    }
```
잘 풀었음.
내가 제시한 솔루션을 보고 코딩해보고 어떻게 이게 동작하는지 (배열 위치가 바뀜)
그리고 이게 어떻게 메모리사용량 측면에서 더 이득인지 생각해보기.